### DISCUSSION TITLE
PLN Open Data

### DISCUSSION THREAD

**Discussion by AlexxNica** [memser-spaceport/pl-network-hub#4] [Ideas]
### What is the idea?

Make it easy for anyone to use data from the Protocol Labs Network (PLN). Data from PLN should be standardized, versioned, documented, and open.

What kinds of information could be open needs to be discussed, but at least some should be available for anyone to use.

Ideally, the more we work on standardizing, versioning, and making data open and accessible, the more we can collaborate, automate processes, and increase data integrity and reliability.

Data sharing can be done through open APIs, spreadsheets, or any other format that is easy to consume programmatically (we can adapt along the way).

**Note:** We would need technical documentation on where to find and how to use those resources.

### Why is this important?

Many teams and applications gather data manually about our ecosystem, which takes time and effort, especially in our fast-paced environment. Inconsistencies also reduce confidence among stakeholders and the public. With information from the network readily available and with a source of truth for PLN-related data, we facilitate these processes, attract and retain talent, and enable developers, analysts, and stakeholders to find, curate, and build upon data that is reliable and up-to-date, increasing innovation and confidence.

Here are a few use cases that benefit from having reliable and open data:
- Teams become more effective at building dashboards and reports from the ecosystem.
- Everyone is equipped with reliable data to use when writing articles, reports, and just sharing in general about the network, its members, orgs, and projects.
- Facilitate the creation of new indexing and searching mechanisms for the network and its members or anyone to use.
- Spaceport teams can use open source contributions to reduce operational costs of maintaining the data while also exploring new use cases and revenue streams.

In summary, content that is easier to find is also easier to use, share, and improve.

**Note:** This idea aligns with the PLN Knowledge Graph project I presented at Launchpad, where I went more in-depth into the topic. If you're interested, you can [watch the presentation on YouTube](https://www.youtube.com/watch?v=Vfm2qEXsoCs).
